js/console.ts

```diff
@@ -90,10 +230,57 @@ function createObjectString(
   return baseString;
 }
 
+function createObjectString(
+  // tslint:disable-next-line:no-any
+  value: any,
+  ...args: [ConsoleContext, number, number]
+): string {
+  if (value instanceof Error) {
+    return value.stack! || "";
+  } else if (Array.isArray(value)) {
+    return createArrayString(value, ...args);
+  } else if (value instanceof Number) {
+    // tslint:disable-next-line:ban-types
+    return createNumberWrapperString(value as Number);
+  } else if (value instanceof Boolean) {
+    // tslint:disable-next-line:ban-types
+    return createBooleanWrapperString(value as Boolean);
+  } else if (value instanceof String) {
+    // tslint:disable-next-line:ban-types
+    return createStringWrapperString(value as String);
+  } else if (value instanceof RegExp) {
+    return createRegExpString(value as RegExp);
+  } else if (value instanceof Date) {
+    return createDateString(value as Date);
+  } else if (value instanceof Set) {
+    // tslint:disable-next-line:no-any
+    return createSetString(value as Set<any>, ...args);
+  } else if (value instanceof Map) {
+    // tslint:disable-next-line:no-any
+    return createMapString(value as Map<any, any>, ...args);
+  } else if (value instanceof WeakSet) {
+    return createWeakSetString();
+  } else if (value instanceof WeakMap) {
+    return createWeakMapString();
+  } else {
```

Comment:
The next line could be inlined for consistency with the other conditions, using `else if`, and `createRawObjectString` inside the `else`.

Reply:
Oh yeah... (I was too tired and totally ignored this...)